src/odemis/gui/comp/overlay/view.py

```diff
@@ -1464,6 +1464,37 @@ def on_left_up(self, evt):
     def draw(self, ctx):
         pass
 
+class StagePointSelectOverlay(PointSelectOverlay):
+    """ Overlay for moving the stage (in physical coordinates) upon the selection of canvas points"""
+
+    def __init__(self, cnvs):
+        PointSelectOverlay.__init__(self, cnvs)
```

Comment:
Why isn't chosen to use super here? Of because PointSelectOverlay is already the parent I think the entire init can be omited